test/unit/cli_test.rb

```diff
@@ -21,8 +21,10 @@ class CliTest < Minitest::Test
     test "#execute_command with the subcommand check starts processing files" do
       violation_message = "This is a violation of code health."
       offense = stub(error?: true, to_s: violation_message)
+      offense.stubs(:is_a?).with(Packwerk::Offense).returns(true)
```

Comment:
> Are we happy with the .stubs(:is_a?).with(X) usage to keep mocks working alongside Sorbet? For classes like Packwerk::Configuration it might require a lot of stubbing to keep them working if we tried to use actual instances

Instead of stubbing `Packwerk::Offense` here, we could use a Fake Test Double, which we can pass and sorbet would be happy. That way we could get rid of this particular stub. The same should hold true for the RunContext and Configuration.

Reply:
Yeah good idea 👍